docs/demo/src/content/tutorial/meta.md

```diff
@@ -0,0 +1,7 @@
+---
+type: tutorial
+title: Basics
```

Comment:
Oh interesting, we also have that property in `overrides/src/content/tutorial/meta.md`. We should remove it from there.

```suggestion
```